libpod/image/parts.go

```diff
@@ -7,12 +7,12 @@ import (
 	"github.com/containers/image/docker/reference"
 )
 
-// imageParts describes the parts of an image's name
-type imageParts struct {
+// Parts describes the parts of an image's name
+type Parts struct {
 	transport   string
-	registry    string
+	Registry    string
 	name        string
-	tag         string
+	Tag         string
```

Comment:
nitpick: could we move public members to the top and private ones down? The public ones need comments.

Reply:
this code is going to be deprecated and removed very very soon